activerecord/lib/active_record/railties/databases.rake

```diff
@@ -365,7 +374,16 @@ db_namespace = namespace :db do
     end
   end
 
-  desc "Creates the database, loads the schema, and initializes with the seed data (use db:reset to also drop the database first)"
+  namespace :setup do
+    task all: ["db:create", :environment, "db:schema:load", :seed]
+
+    ActiveRecord::Tasks::DatabaseTasks.for_each(databases) do |name|
+      desc "Creates the #{name} database, loads the schema, and initializes with the seed data (use db:reset:#{name} to also drop the database first)"
+      task name => ["db:create:#{name}", :environment, "db:schema:load:#{name}", "db:seed"]
```

Comment:
Will seed try to load seeds on all the dbs or just the named ones here?

Reply:
Thanks for reviewing @eileencodes! This will unfortunately seed all dbs. From my understanding of `db/seeds.rb`, it can reference any ActiveRecord model and there's no scope to which database will be touched. I don't think there's a way to enforce that a seed file will only touch a single database, but I could be mistaken.